lib/fs.js

```diff
@@ -2379,8 +2379,13 @@ WriteStream.prototype.open = function() {
 
 
 WriteStream.prototype._write = function(data, encoding, cb) {
-  if (!(data instanceof Buffer))
-    return this.emit('error', new Error('Invalid data'));
+  if (!(data instanceof Buffer)) {
+    const err = new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                                     'data',
+                                     ['string', 'Buffer'],
```

Comment:
The type check only checks if the data is a buffer, can you change this to `'Buffer'` instead?

Reply:
Oops! I was looking at the usage elsewhere in the same file and missed this. Thanks, I fixed it! 👍